Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -142,19 +142,29 @@ public void Disconnect()
         {
```

Comment:
Make this method virtual,  and override in ULocalConnectionToClient

Ideally this method does not have special case depending on connection id

Reply:
Given the open debate about whether ULocalConnectionToClient will continue to exist, I think it's better not to broaden the scope of this PR.  Baby steps, as you all say.